scripts/set_version.py

```diff
@@ -15,50 +33,25 @@ def get_parser():
     return parser
 
 
-def set_version(version):
+def fill_template(fn, version):
```

Comment:
```suggestion
def fill_quarto_template(fn, version):
```

Reply:
This is a generic filler, called as `    fill_template(QUARTO_TEMPLATE, args.version)`, so think it's OK as-is.